### PR TITLE
enhancement: adds Transak loading timeout fallback

### PR DESCRIPTION
**Pull request by jeeanribeiro**
## Summary

Some users have reported that their Transak widget is not loading and that could be due to how loading is detected, this PR adds a fallback timeout of 5 seconds if the observed changes are not detected.

### Platforms

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [x] Windows

## Checklist

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [x] I have verified that new and existing unit tests pass locally with my changes
-   [x] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/lib/electron/preloads/transak.preload.ts [hunk 1]
```diff
@@ -20,6 +21,10 @@ function observeStyleChanges(targetNode: Element): { disconnect: () => void } {
     const observer = new MutationObserver(callback)
     observer.observe(targetNode, config)
 
+    const fallbackTimeout = setTimeout(() => {
+        observer.disconnect()
+    }, 5000)
+
     return {
         disconnect: () => observer.disconnect(),
     }
```
Comment (Tuditi) [inline packages/desktop/lib/electron/preloads/transak.preload.ts]: Why are we returning this function?

Comment (jeeanribeiro) [reply packages/desktop/lib/electron/preloads/transak.preload.ts]: So the caller could also control the disconnection, but it is not needed, will remove it

File: packages/desktop/lib/electron/preloads/transak.preload.ts [hunk 2]
```diff
@@ -20,9 +21,9 @@ function observeStyleChanges(targetNode: Element): { disconnect: () => void } {
     const observer = new MutationObserver(callback)
     observer.observe(targetNode, config)
 
-    return {
-        disconnect: () => observer.disconnect(),
-    }
+    const fallbackTimeout = setTimeout(() => {
+        observer.disconnect()
```
Comment (Tuditi) [inline packages/desktop/lib/electron/preloads/transak.preload.ts:25]: Commenting out l.13-l15 doesn't result in any improvement. Maybe you need to add `void ipcRenderer.invoke('transak-loaded')` to the callback?

**Review by Tuditi — CHANGES_REQUESTED** [review]

File: packages/desktop/lib/electron/preloads/transak.preload.ts (hunk 2 above)
Comment (jeeanribeiro) [reply packages/desktop/lib/electron/preloads/transak.preload.ts:25]: 🤦 how could I forget it 🤣

**Review by Tuditi — APPROVED** [review]
LGTM!

**Pull request merged**